exporter/loggingexporter/logging_exporter.go

```diff
@@ -38,7 +39,19 @@ type loggingExporter struct {
 }
 
 func (s *loggingExporter) pushTraces(_ context.Context, td pdata.Traces) error {
-	s.logger.Info("TracesExporter", zap.Int("#spans", td.SpanCount()))
+	fields := []zap.Field{
+		zap.Int("#spans", td.SpanCount()),
+	}
+
+	ac := configauth.ExtractPDataContext(td.ResourceSpans().At(0).PDataContext())
+	if ac != nil {
+		fields = append(fields, zap.String("sub", ac.Subject()))
+		fields = append(fields, zap.Strings("groups", ac.Groups()))
+	} else {
+		fields = append(fields, zap.String("auth", "none"))
+	}
```

Comment:
This should be probably moved to `MarshalXXX` implementations and should log PDataContext from each ResourceSpan, not just the first one.
We can make PDataContext serializable or stringifyable for logging purposes or give it a a `Log` method.

Reply:
Done.